ui-core/src/components/inputs/TolerancePicker.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useRef } from 'react';
+import Input, { InputProps } from './Input';
+import Modal from '../Modal';
+
+const TolerancePicker: React.FC<InputProps> = (inputProps) => {
+  const [showPicker, setShowPicker] = useState(false);
+  const [minusTolerance, setMinusTolerance] = useState('00');
+  const [plusTolerance, setPlusTolerance] = useState('00');
+  const [inputValue, setInputValue] = useState('');
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const handleTimeChange = (newMinusTolerance: string, newPlusTolerance: string) => {
+    setMinusTolerance(newMinusTolerance);
+    setPlusTolerance(newPlusTolerance);
+    const formattedValue = `-${newMinusTolerance}/+${newPlusTolerance}`;
+    setInputValue(formattedValue);
```

Comment:
The parent component should give a onChange function, which should be call here. It should take a MinusTolerance and a PlusTolerance which should be both in seconds:
```
onChange: (minusTolerance: number, plusTolerance: number ) => void
```